src/plugins/expressions/common/service/expressions_services.ts

```diff
@@ -289,7 +312,7 @@ export class ExpressionsService
   }
 
   private isStarted(): boolean {
-    return !!(ExpressionsService.started.has(this) || this.parent?.isStarted());
+    return !!ExpressionsService.started.has(this);
```

Comment:
There is no need to cast it anymore. That was here because of the optional chaining operator which is no longer here.

```suggestion
    return ExpressionsService.started.has(this);
```